superset/__init__.py

```diff
@@ -19,7 +19,6 @@
 from werkzeug.local import LocalProxy
 
 from superset.app import create_app
-from superset.connectors.connector_registry import ConnectorRegistry
 from superset.extensions import (
```

Comment:
you're going to want to add the datasourceDao in here so that you can reference it later like `superset.DatasourceDAO`